weblogic-azure-aks/src/main/bicep/modules/setupDBConnection.bicep

```diff
@@ -65,6 +65,8 @@ param identity object = {}
 
 @description('JNDI Name for JDBC Datasource')
 param jdbcDataSourceName string = 'jdbc/contoso'
+@description('tags for the resources')
+param tagsByResource object = {}
```

Comment:
To pass the new tag pipeline, no need to refresh the offer in partner center.

Reply:
Will do.